src/tile.h

```diff
@@ -113,17 +112,28 @@ class TileItemVector : private ItemVector
 	uint16_t downItemCount = 0;
 };
 
-class Tile : public Cylinder
+class Tile : public Thing
 {
 public:
-	static Tile& nullptr_tile;
+	static std::shared_ptr<Tile> nullptr_tile;
```

Comment:
Could this be `const`
```suggestion
	static const std::shared_ptr<Tile> nullptr_tile;
```